beard/clustering/wrappers.py

```diff
@@ -83,22 +83,27 @@ def __init__(self, method="single", affinity="euclidean",
             - "affinity": for passing an affinity matrix Xa;
             - None: for not passing anything but the labels.
 
-        :param scoring: callable
+        :param supervised_scoring: callable or None
             The scoring function to maximize in order to estimate the best
-            threshold. There are 6 possible cases based on the availability of
-            ground truth data passed in `y` and the value of `scoring_data`:
-                - Ground truth data and scoring_data == "raw":
-                    scoring(X, label_true, label_pred);
-                - No ground truth data and scoring_data == "raw":
-                    scoring(X, label_pred);
-                - Ground truth data and scoring_data == "affinity":
-                    scoring(Xa, label_true, label_pred);
-                - No ground truth data and scoring_data == "affinity":
-                    scoring(Xa, label_pred);
-                - Ground truth data and scoring_data is None:
-                    scoring(label_true, label_pred);
-                - No ground truth data and scoring_data is None:
-                    scoring(label_pred).
+            threshold. Ground truth must be True for this scoring function.
```

Comment:
" Ground truth must be True " does not actually mean anything. We should rather say: "If labels are provided in `y`, ..."